src/plugins/hetero/executable_network.cpp

```diff
@@ -58,12 +63,22 @@ using namespace InferenceEngine::HeteroConfigParams;
 template <typename T>
 using NodeMap = std::unordered_map<ngraph::Node*, T>;
 
+namespace {
+ov::AnyMap any_copy(const Configs& params) {
+    ov::AnyMap result;
+    for (auto&& value : params) {
+        result.emplace(value.first, value.second);
+    }
+    return result;
+}
+}
+
+
 HeteroExecutableNetwork::HeteroExecutableNetwork(const InferenceEngine::CNNNetwork& network,
                                                  const Configs& user_config,
-                                                 Engine* plugin)
+                                                 const ov::hetero::Plugin* plugin)
```

Comment:
Maybe it is better to use `const std::shared_ptr<const ov::IPlugin>& plugin`

Reply:
Done, but it's temporary solution to link old `HeteroExecutableNetwork` with new `Plugin`